Vagrantfile

```diff
@@ -172,9 +172,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     vb.customize ['modifyvm', :id, '--nictype1', 'virtio']
   end
 
-  # Use vagrant-vbguest plugin to make sure Guest Additions are in sync
-  config.vbguest.auto_reboot = true
-  config.vbguest.auto_update = true
+  # Disable auto-update as it causes plugin failures
```

Comment:
It would be nice to specify what failures `auto_update` can cause.

Additionally, [from the docs](https://github.com/dotless-de/vagrant-vbguest#running-as-a-middleware):

> You may switch off the middleware by setting the vm's config vbguest.auto_update to false. This is a per box setting. On multi vm environments you need to set that for each vm.

So disabling `auto_update` will put us in "command mode"? So, VirtualBox Guest Additions won't be installed unless we call vbguest manually?

Reply:
> It would be nice to specify what failures auto_update can cause.

Well, not the auto-update causes directly the plugin failures but the fact of updating the plugin. Hence, it is really hard to say what will be broken per setup as both VirtualBox and Vagrant can raise issues. Last time I had three different kind of issues because of this plugin.

> So disabling auto_update will

I believe the documentation is not phrased well, as it is not switching off the `vbguest` middleware -- I've seen it running, just not updates itself.

Reply:
> Well, not the auto-update causes directly the plugin failures but the fact of updating the plugin.

Sounds like you expect this flag to trigger auto-plugin updates, but I think "auto-update" is supposed to mean "update the VM with the latest GuestAdditions automatically," rather than "updating the vbguest plugin itself automatically". Now, I would have to dig into Vagrant and vbguest more deeply to give you feedback on the impact of disabling this. Since I've provisioned the VM on my end successfully, I'll stay on the side of "it is working" against "let's make it right". :+1: